<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDialog</class>
 <widget class="QDialog" name="DeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Üye Sil</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="tc_input">
     <property name="inputMask">
      <string>99999999999</string>
     </property>
     <property name="placeholderText">
      <string>TC Kimlik No</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_uye_sil">
     <property name="text">
      <string>Üyeyi Sil</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
